tests/bluetooth/tester/btp_spec.txt

```diff
@@ -1422,3 +1422,29 @@ Events:
 		Event parameters:	<none>
 
 		This event indicate that node was provisioned.
+
+	Opcode 0x84 - Link open Event
+
+		Controller Index:	<controller id>
+		Event parameters:	Bearer (1 octet)
+
+		Bearer is a bitmask with the following available bits:
```

Comment:
I'm not sure it's right to say this is a bitmask, since only one of the values is valid at a time. This is different from the bt_mesh_prov_enable() API which does accept combinations of multiple bearers.

Reply:
Yes, I just copied this from header. But here only one bearer is valid. Done